MediaRecorder.html

```diff
@@ -220,63 +222,87 @@
             <code>start()</code> method is invoked, the UA MUST run the
             following steps:
             <ol>
-              <li>Let <var>timeslice</var> be the method's first argument, if
-              provided, or <code>undefined</code>.</li>
-              <li>If <code>state</code> is not <code>inactive</code>, throw an
-              <code>InvalidStateError</code> DOMException and abort these
-              steps.</li>
+              <li>Let <var>target</var> be the MediaRecorder context object.
+              Let <var>timeslice</var> be the method's first argument, if
+              provided, or <code>undefined</code>. </li>
+
+              <li>If <a>state</a> is not <a href="#dom-recordingstate-inactive">
+              inactive</a>, throw an <code>InvalidStateError</code> DOMException
+              and abort these steps.</li>
+
               <li>If the stream's isolation properties disallow access from
               this MediaRecorder, throw a <code>SecurityError</code>
               DOMException and abort these steps.</li>
-              <li>Set <code>state</code> to "recording", and run the following
-              steps in parallel:
-                <ol>
 
+              <li>Set <a>state</a> to <a href="#dom-recordingstate-recording">
+              recording</a>, and run the following steps in parallel:
+                <ol>
                   <li>Once media becomes available from one or more of the
-                  stream's tracks, start gathering the data into a Blob (see
-                  [[!FILE-API]]), and queue a task, using the DOM manipulation
-                  task source, to fire an event named <code>start</code>.</li>
+                  stream's tracks, start gathering the data into a Blob
+                  ([[!FILE-API]]) <var>blob</var> and queue a task, using the
+                  DOM manipulation task source, to <a
+                  href="https://dom.spec.whatwg.org/#concept-event-fire">fire
+                  an event</a> named <code>start</code>.</li>
```

Comment:
This and other cases elsewhere still need "at *target*". I'm just imitating https://dom.spec.whatwg.org/#firing-events-example FWIW.